arch/arm/core/aarch32/mpu/arm_mpu.c

```diff
@@ -39,36 +39,12 @@ LOG_MODULE_DECLARE(mpu);
  */
 static uint8_t static_regions_num;
 
-/**
- *  Get the number of supported MPU regions.
- */
-static inline uint8_t get_num_regions(void)
```

Comment:
This is also refactoring. I am seeing this is now moved to the internal headers, as well. Can we move this to the refactoring PR?

Reply:
Any chance we can leave this one alone?  Pulling this out into a separate commit is going to be pretty tricky.